administrator/components/com_menus/layouts/joomla/searchtools/default.php

```diff
@@ -1,6 +1,6 @@
 <?php
 /**
- * @package     Joomla.Administrator
+ * @package     Joomla.Site
```

Comment:
Please revert this change
```suggestion
 * @package     Joomla.Administrator
```